docs/pages/text.js

```diff
@@ -41,63 +46,85 @@ export default function TextPage({ generatedDocGen }: {| generatedDocGen: DocGen
+    <Text color="default" size="400">Default</Text>
+    <Text color="success" size="400">Success</Text>
+    <Text color="warning" size="400">Warning</Text>
+    <Text color="error" size="400">Error</Text>
+    <Text color="shopping" size="400">Shopping</Text>
+    <Box color="primary" padding={1}>
+      <Text color="light" size="400">Light</Text>
+    </Box>
+    <Box color="infoWeak" padding={1}>
+      <Text color="dark" size="400">Dark</Text>
+    </Box>
+  </Flex>
+          `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Overflow"
+          description="Gestalt provides utility options to deal with text overflow."
+        >
+          <Example
+            name="overflow"
+            showHeading={false}
+            defaultCode={`
+<Box borderStyle="sm" maxWidth={200} padding={1}>
+  <Flex gap={2} direction="column">
 
-  <Text weight="bold">normal:</Text>
-  <Text overflow="normal">
-    This is a long and Supercalifragilisticexpialidocious sentence.
-    次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉&#39;
-  </Text>
+    <Text weight="bold">breakWord (default):</Text>
+    <Text>
+      This is a long and Supercalifragilisticexpialidocious sentence.
+      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉
+    </Text>
 
-  <Text weight="bold">noWrap:</Text>
-  <Text overflow="noWrap">
-    This is a long and Supercalifragilisticexpialidocious sentence.
-  </Text>
+    <Text weight="bold">normal:</Text>
+    <Text overflow="normal">
+      This is a long and Supercalifragilisticexpialidocious sentence.
+      次の単語グレートブリテンおよび北アイルランド連合王国で本当に大きな言葉&#39;
+    </Text>
 
-  <Text weight="bold">lineClamp:</Text>
-  <Text lineClamp={2}>
-    This is a long and Supercalifragilisticexpialidocious sentence.
-  </Text>
-</Flex>
-`}
-      />
-      <Example
-        description={`
-    You can apply \`size\` options to define the size of the text. These font sizes follow those available through our [Design Tokens](/design_tokens#Font-size). If your text needs to be a [semantic heading (H1-H6)](/https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements), use [Heading](/heading) instead.
+    <Text weight="bold">noWrap:</Text>
+    <Text overflow="noWrap">
+      This is a long and Supercalifragilisticexpialidocious sentence.
+    </Text>
+
+    <Text weight="bold">lineClamp:</Text>
+    <Text lineClamp={2}>
+      This is a long and Supercalifragilisticexpialidocious sentence.
+    </Text>
+  </Flex>
+</Box>
+        `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Sizes"
+          description={`
+          You can apply \`size\` options to define the size of the text. These font sizes follow those available through our [Design Tokens](/design_tokens#Font-size). If your text needs to be a [semantic heading (H1-H6)](/https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements), use [Heading](/heading) instead.
```

Comment:
ALERT link  is broken

Reply:
[semantic heading (H1-H6)](**/**https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)